<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Recent publications</title>

<script src="site_libs/header-attrs-2.16/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Jinzhuang Dou</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">About me</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="teaching.html">Conferences and Teaching</a>
</li>
<li>
  <a href="links.html">Links</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Recent publications</h1>

</div>


<p>You can find all publication on <a
href="https://scholar.google.com/citations?user=2_q_kTUAAAAJ&amp;hl=en">google
scholar</a></p>
<p align="center">
<img src="figs/paper.jpg" width="1000" />
</p>
<div id="firstco-first-author" class="section level2">
<h2>First/Co-first author</h2>
<ul>
<li><p><a
href="https://www.nature.com/articles/s41587-023-01873-x">Monopogen:
single nucleotide variant calling from single cell sequencing.</a> <br/>
<strong>J Dou</strong>, Y Tan, KH Kock, J Wang , X Cheng, LM Tan, KY
Han, CC Hon, WY Park, JW Shin, H Jin, H Chen, L Ding, S Prabhakar, N
Navin. K Chen. <strong>Nature Biotechnology</strong> 2023.<br />
see <a
href="https://www.nature.com/articles/s41588-023-01544-2">Research
Highlight in Nature Genetics.</a></p></li>
<li><p><a
href="https://www.cell.com/cell/pdf/S0092-8674(19)31070-0.pdf">Large-scale
whole-genome sequencing of three diverse Asian populations in
Singapore.</a> <br/> D Wu*, <strong>J Dou*</strong>, X Chai*, C Bellis,
A Wilm, C.C. Shih, W.W.J Soon, N Bertin, C.B Lin, C.C Khor, M DeGiorgio,
S.S Cheng, L Bao, N Karnani, W Hwang, S Davila, P Tan, A Shabbir, A Moh,
E Tan, J.N Foo, L.L Goh, K.P Leong, R.S.Y Foo, C Lam, A Richards, C.Y
Cheng, T Aung, T Wong, H Ng, M Ackers-Johnson, E Aliwarga, K Kim Ban, D
Bertrand, J Chambers, D Hui Chan, C Li Chan, M Chee, M Chee, P Chen, Y
Chen, E Chew, W Chew, L Chiam, J Chong, I Chua, S Cook, W Dai, R
Dorajoo, C Foo, R Goh, A Hillmer, I Irwan, F Jaufeerally, A Javed, J
Jeyakani, J Koh, J Koh, P Krishnaswamy, J Kuan, N Kumari, A Lee, S Lee,
S Lee, Y Lee, S Leong, Z Li, P Li, J Liew, O Liew, S Lim, W Lim, C Lim,
T Lim, C Lim, S Loh, A Lok, C Chin, S Majithia, S Maurer-Stroh, W Meah,
S Mok, N Nargarajan, P Ng, S Ng, Z Ng, J Ng, E Ng, S Ng, S Nusinovici, C
Ong, B Pan, V Pedergnana, S Poh, S Prabhakar, K Prakash, I Quek, C
Sabanayagam, W See, Y Sia, X Sim, W Sim, J So, D Soon, E Tai, N Tan, L
Tan, H Tan, W Tan, M Tandiono, A Tay, S Thakur, Y Tham, Z Tiang, G Toh,
P Tsai, L Veeravalli, C Verma, L Wang, M Wang, W Wong, Z Xie, K Yeo, L
Zhang, W Zhai, Y Zhao, J Liu, C Wang. <strong>Cell</strong> 2019
(<em>Cover story</em>)</p></li>
<li><p><a
href="https://www.science.org/doi/full/10.1126/sciadv.add6997">Metabolic
competition is an important driver of tumor resistance after CAR NK cell
therapy and can be overcome by cytokine engineering.</a> <br/> L Li*, V
Mohanty*, <strong>J Dou*</strong>, Y Huang*, P P. Banerjee, Q Miao, J
Lohr, T Vijaykumar, B Knoechel, R Basar, S Liang, M Kaplan, M Daher, E
Liu, L Muniz-Feliciano, T J. Laskowski, D Marin, S Mielke, R E. C,
Elizabeth J. Shpall, K Chen, K Rezvani. <strong>Science Advance</strong>
2023</p></li>
<li><p><a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02679-x">Bi-order
multimodal integration of single-cell data.</a> <br/> <strong>J
Dou*</strong>, S Liang*, V Mohanty, X Cheng, S Kim, J Choi, Y Li, K
Rezvani, R Chen, K Chen. <strong>Genome Biology</strong> 2022</p></li>
<li><p><a
href="https://academic.oup.com/bib/article/22/3/bbaa084/5857014">Using
off-target data from whole-exome sequencing to improve genotyping
accuracy, association analysis, and polygenic risk prediction.</a> <br/>
<strong>J Dou*</strong>, D Wu*, L Ding, K Wang, M Jiang, X Chai, D.
Reilly, E Tai, J Liu, X Sim, S Cheng, C Wang. <strong>Brief in
Bioinformatics</strong> 2020</p></li>
<li><p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007021">Estimation
of kinship coefficient in structured and admixed populations using
sparse sequencing data.</a> <br/> <strong>J Dou</strong>, B Sun, X Sim,
I Irwan, J D Hughes, D F Reilly, E Tai, J Liu, C Wang. <strong>PLoS
Genetics</strong> 2017</p></li>
<li><p><a
href="https://academic.oup.com/genetics/article/206/3/1237/6064258">Whole-genome
restriction mapping by “subhaploid”-based RAD sequencing: an efficient
and flexible approach for physical mapping and genome scaffolding.</a>
<br/> <strong>J Dou</strong>, H Dou, C Mu, L Zhang, Y Li, J, T Li, X Hu,
S Wang, Z Bao. <strong>Genetics</strong> 2017 (<em>Cover
story</em>)</p></li>
<li><p><a href="https://www.nature.com/articles/srep19244">Evaluation of
the 2b-RAD method for genomic selection in scallop breeding.</a> <br/>
<strong>J Dou</strong>, X Li, Q Fu, W Jiao, Y Li, T Li, Y Wang, X Hu, S
Wang, Z Bao. <strong>Scientific Report</strong> 2016</p></li>
<li><p><a
href="https://academic.oup.com/dnaresearch/article/21/1/85/347657">High-resolution
linkage and quantitative trait locus mapping aided by genome survey
sequencing: building up an integrative genomic framework for a bivalve
mollusc.</a> <br/> W Jiao*, X Fu*, <strong>J Dou*</strong>, H Li, H Su,
J Mao, Q Yu, L Zhang, X Hu, X Huang, Y Wang, S Wang, Z Bao. <strong>DNA
Research</strong> 2013 (<em>Editor’s choice</em>)</p></li>
<li><p><a
href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0079960">RADtyping:
an integrated package for accurate de novo codominant and dominant RAD
genotyping in mapping populations.</a> <br/> X Fu*, <strong>J
Dou*</strong>, J Mao, H Su, W Jiao, L Zhang, X Hu, X Huang, S Wang, Z
Bao. <strong>PLoS ONE</strong> 2013</p></li>
<li><p><a
href="https://biologydirect.biomedcentral.com/articles/10.1186/1745-6150-7-17">Reference-free
SNP calling: improved accuracy by preventing incorrect calls from
repetitive genomic regions.</a> <br/> <strong>J Dou</strong>, X Zhao, X
Fu, W Jiao, N Wang, L Zhang, X Hu, S Wang, Z Bao. <strong>Biology
Direct</strong> 2012</p></li>
</ul>
</div>
<div id="contributing-author-selected" class="section level2">
<h2>Contributing author (selected)</h2>
<ul>
<li><p><a
href="https://biologydirect.biomedcentral.com/articles/10.1186/1745-6150-7-17">Cord
blood characteristics determine the efficacy of allogeneic IL-15 armored
CAR-transduced natural killer cells against CD19-positive B cell
tumors.</a> <br/> D Marin*, Y Li*, R Basar*, H Rafei*, M Daher*,
<strong>J Dou</strong>, V Mohanty, M Dede, Y Nieto,N Uprety,S Acharya,E
Liu ,J Wilson, P Banerjee, H Macapinlac, C Ganesh, P Thall, R Bassett, M
Ammari, S Rao, Cao,M Shanley, C Hosing,P Kebriaei, L Nastoupil, C
Flowers, S Moseley,P Lin, S Ang, U Popat, M Qazilbash,R Champlin, K
Chen, E Shpall, K Rezvani. <strong>Nature Medicine</strong>
2023</p></li>
<li><p><a href="https://genesdev.cshlp.org/content/36/13-14/822">KLF5
governs sphingolipid metabolism and barrier function of the skin.</a>
<br/> Y Lyu, Y Guan, L Deliu, E Humphrey, J Frontera, Y Yang, <strong>J
Dou</strong>,…,K Chen,P Nagarajan, Y Ge. <strong>Genes
Development</strong> 2022</p></li>
<li><p><a
href="https://www.nature.com/articles/s41467-022-28028-x">Systematic
decomposition of sequence determinants governing CRISPR/Cas9
specificity.</a> <br/> R Fu*, W He*, <strong>J Dou</strong>, O D.
Villarreal, E Bedford, H Wang, C Hou, L Zhang, Y Wang, Y Chen, X Gao, M
Depken, H Xu. <strong>Nature Communications</strong> 2022</p></li>
<li><p><a
href="https://www.nature.com/articles/s43588-021-00070-7">Single-cell
manifold-preserving feature selection for detecting rare cell
populations.</a> <br/> S Liang, V Mohanty, <strong>J Dou</strong>, Q
Miao, Y Huang, M Muftuoglu, L Ding, W Peng, K Chen. <strong>Nature
Computational Science</strong> 2021</p></li>
<li><p><a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02291-5">Single-cell
copy number lineage tracing enabling gene discovery.</a> <br/> F Wang, Q
Wang, V Mohanty, S Liang, <strong>J Dou</strong>, J Han, D Conterno
Minussi, R Gao, L Ding, N Navin, K Chen. <strong>Genome Biology</strong>
2021</p></li>
<li><p><a
href="https://academic.oup.com/mbe/article/38/10/4463/6307269?login=true">Genetic
Admixture in the Culturally Unique Peranakan Chinese Population in
Southeast Asia.</a> <br/> D Wu, P Li, B Pan, Z Tiang, <strong>J
Dou</strong>, I Williantarra, A Pribowo, R Nurdiansyah, SG Peranakan
Project, R SY Foo, C Wang. <strong>Molecular Biology and
Evolution</strong> 2021</p></li>
<li><p><a
href="https://www.cell.com/cell-reports/pdfExtended/S2211-1247(21)00849-4">Generation
of glucocorticoid-resistant SARS-CoV-2 T cells for adoptive cell
therapy.</a> <br/> R Basar, N Uprety, E Ensley, M Daher, K
Klein,…,<strong>J Dou</strong>,..,K Rezvani. <strong>Cell
Report</strong> 2021</p></li>
<li><p><a href="https://www.jci.org/articles/view/142116">Targeting the
v integrin-TGF-beta axis improves natural killer cell function against
glioblastoma stem cells.</a> <br/> H Shaim, M Shanley, R Basar, M
Daher,…, <strong>J Dou</strong>,…,K Chen, F Lang, E Shpall, A
Heimberger, K Rezvan. <strong>The Journal of Clinical
Investigation</strong> 2021</p></li>
<li><p><a href="https://www.nature.com/articles/s41467-019-12489-8">De
novo identification of essential protein domains from CRISPR-Cas9
tiling-sgRNA knockout screens.</a> <br/> W He, L Zhang, O Villarreal, R
Fu, E Bedford, <strong>J Dou</strong>, A Patel, M Bedford, X Shi, T
Chen, B Bartholomew, H Xu. <strong>Nature Communications</strong>
2019</p></li>
<li><p><a
href="https://www.nature.com/articles/s41467-017-01927-0">Scallop genome
reveals molecular adaptations to semi-sessile life and neurotoxins.</a>
<br/> Y Li, X Sun, X Hu, X Xun, J Zhang, X Guo,…, <strong>J
Dou</strong>,…, Z Jiang, R Li, S Wang, Z Bao. <strong>Nature
Communications</strong> 2017</p></li>
<li><p><a href="https://www.nature.com/articles/s41559-017-0120">Scallop
genome provides insights into evolution of bilaterian karyotype and
development.</a> <br/> S Wang, J Zhang, W Jiao, J Li, X Xun, Y
Sun,…,<strong>J Dou</strong>,…, Z Jiang, D Chourrout, R Li, Z Bao.
<strong>Nature Ecology and Evolution</strong> 2017</p></li>
</ul>
<p>* denotes equal contributions, <sup>#</sup> corresponding authors</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
